bsp_diff/caas/device/intel/civ/host/vm-manager/0046-Multi-Android-VM-support.patch

```diff
@@ -0,0 +1,457 @@
+From a002c05cb8ce6537888d218d3d083b9e54b62d21 Mon Sep 17 00:00:00 2001
+From: "Suresh, Prashanth" <prashanth.suresh@intel.com>
+Date: Thu, 15 Jun 2023 12:13:48 +0530
+Subject: [PATCH] Multi Android VM support
+
+Tracked-On: OAM-110639
+
+Signed-off-by: Suresh, Prashanth <prashanth.suresh@intel.com>
+---
+ scripts/create_vms.sh                     |  70 +++++++++++++
+ scripts/setup_civ_ini.sh                  |  47 +++++++--
+ scripts/setup_host.sh                     |  67 +++++++++++-
+ scripts/setup_host_multiple_android_vm.sh | 118 ++++++++++++++++++++++
+ scripts/start_flash_usb.sh                |  22 +++-
+ 5 files changed, 310 insertions(+), 14 deletions(-)
+ create mode 100644 scripts/create_vms.sh
+ create mode 100644 scripts/setup_host_multiple_android_vm.sh
```

Comment:
There are 2 new files:
- create_vms.sh
- setup_host_multiple_android_vm.sh

Why can't we combine them into a single script file?
Is create_vms.sh reused by setup_host.sh?